drivers/adc/ad469x/ad469x.c

```diff
@@ -628,6 +630,7 @@ int32_t ad469x_init(struct ad469x_dev **device,
 	dev->capture_data_width = init_param->capture_data_width;
 	dev->dev_id = init_param->dev_id;
 	dev->dcache_invalidate_range = init_param->dcache_invalidate_range;
+	dev->ch_sequence = AD469x_standard_seq;
```

Comment:
nitpick: naming of `AD469x_standard_seq` is a bit odd as it mixes upper and lower case letters.